.github/workflows/build.yml

```diff
@@ -67,18 +73,19 @@ jobs:
   instrumentation-tests:
     name: Instrumentation tests
     runs-on: macOS-latest
-    timeout-minutes: 60
+    timeout-minutes: 30
```

Comment:
Are we definitely always below 30 minutes?

Reply:
15 mins in general